lib/intelligent_foods/serializers/recipient_serializer.rb

```diff
@@ -14,7 +14,7 @@ def to_json
         email: email,
         phone: phone,
         delivery_instructions: delivery_instructions,
-      }
+      }.delete_if { |_k, v| v.blank? }
```

Comment:
move this into a method and give it a name that describes what it's actually doing